components/atoms/button/components/button/button.js

```diff
@@ -32,7 +32,7 @@ const StyledButton = styled.button`
     'box-shadow: 0px 10px 24px 0px rgba(0, 0, 0, .12), 0px 10px 24px 0px rgba(0, 0, 0, .12), 0px 10px 24px 0px rgba(0, 0, 0, .12);'}
   cursor: pointer;
   display: ${({ block, centre }) => (centre || block ? 'block' : 'inline-block')};
-  font-family: ${({ theme }) => theme.font};
+  font-family: ${({ theme }) => theme.font};   /* stylelint-disable-line */
```

Comment:
@majid-amiri Not a fan of linting disabling - what was the error here?

Reply:
There was no choice other than disable stylelint for this line because it was complaining about a "generic font family" and we're using variables here, so we should either disable "font-family-no-missing-generic-family-keyword" totally or disable it in places like this one.